keyboards/sepsepkbd/yubino_hara/keyboard.json

```diff
@@ -0,0 +1,111 @@
+{
+    "manufacturer": "t-ando",
+    "keyboard_name": "yubino_hara",
+    "maintainer": "nagmeal",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "rgblight": true,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "split": {
+        "enabled": true,
+        "soft_serial_pin": "GP0",
+        "transport": {
+            "sync": {
+                "matrix_state": true
+            }
+        },
+        "usb_detect": {
+            "enabled": true
+        }
```

Comment:
This isn't needed.  Split usb detect is the default for ARM boards, like the RP2040. 
```suggestion
```

Reply:
I understand. Thank you.